src/posts/blog/7-budget/budget.ts

```diff
@@ -0,0 +1,90 @@
+import podOverviewChart from './img/pod-overview-chart.png'
+import { ImageMeta } from 'posts/post'
+
+const image: ImageMeta = {
+  url: podOverviewChart,
+  width: 1632,
+  height: 502,
+}
+
+const title =
+  'How our house got our lives back by using a collective “risk budget”'
+
+const author = 'Jeremy Blanchard'
+const date = 'February 7, 2021'
+
+const summary =
+  'Navigating house agreements around COVID can be really stressful. We made a tool to help make it a lot easier.'
+const content = `
+
+<style>
+img {
+  max-width: 100%;
+  height: auto;
+}
+</style>
+
+
+
+*Today we’re launching a new version of the microCOVID Risk Tracker with features for households and pods to collectively budget their risk over time.* [*You can download the latest version of the Risk Tracker here →*](/tracker)
+
+
+<figure>
+  <img src="${podOverviewChart}" alt="Graph of 4 housemate's risk budget, showing risk points used and risk points remaining." />
+</figure>
+
+
+## How the Risk Tracker helped our house get our lives back
+
+Here’s a bit of the backstory behind this new version. Up until [microCOVID](/) was released in August 2020, my house was struggling with the same thing many communal houses were struggling with: we want to make sure to keep ourselves, each other, and the wider community safe from COVID, *and* at the same time, there are things that are important enough that we want to be able to find a way to do them. 
+
+The dilemma: how do you make decisions about what a “safe amount of risk is” when living with many other people?
+
+At that time our system was similar to many other group houses: masked, outdoor, distance hangs were okay, but anything other than that needed group input/consensus. As you might imagine, having every single person’s activities be subject to a consensus discussion in a house meeting is a recipe for a ton of stress and struggle. (“What’s riskier: a 3-hour unmasked. outdoor hang? Or a 30-minute indoor, masked hang? 🤷‍♂️” and “Okay I know you really want to see your bodyworker and I really want to see my partner, but are either of those risks we’re willing to take?!”)
+
+So when I got my hands on the original microCOVID spreadsheet, I went to town upgrading it for our house to use. I asked the team dozens of questions to make sure I had my head around how it was supposed to work.
```

Comment:
I think this needs a transition between your house's struggle and you upgrading the sheet.

Perhaps something along the lines of, "When I saw the original microCOVID spreadsheet, I realized it had the potential to completely change how we had these discussions. If we could quantify and track how much risk each of us were taking, we could allocate everyone a small but even share of risk that they could use to make their own decisions, without consulting the house every time.

The original microCOVID spreadsheet didn't have features for a pod to share risk like this , so I went to town upgrading it for our house to use...

Reply:
Well put!